### ISSUE TITLE
Enhance sample app to show traces and metrics

### ISSUE THREAD

**Issue by rohitjoshi** [SigNoz/sample-rust-app#1]
Thanks for creating this sample app. I did try to use but it doesn't populate the method or status code. Can you please add the support? 
Also if you can show the metrics e.g. number of requests, status code etc.

**Comment by pranay01** [comment]
thanks for opening the issue @rohitjoshi 

Which version of SigNoz are you using? And what type of service are you trying to monitor - HTTP/GRPC?

**Comment by rohitjoshi** [comment]
Version: v0.8
Application : Both HTTP (axum) and gRPC(tonic)
Opentelemetry transport: gRPC/ tonic

**Comment by frol** [comment]
I experience the same issue here. SigNoz always reports 0.01ms duration even when it is noticeable that it takes seconds to perform the query, and there are no useful traces and HTTP method/status info.

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/304265/176950147-495e7e9b-b38b-4f11-9a8c-e36d908c4a5c.png">

SigNoz version: 0.9.1 (running in standalone Docker mode on MacOS arm64)

**Comment by pranay01** [comment]
@frol Are you running the sample app ?

**Comment by frol** [comment]
@pranay01  yep, I’m running this sample app and trying to understand how I should setup opentelemetry before I go implementing it in my production apps

**Comment by srikanthccv** [comment]
@frol What HTTP framework are you using?

**Comment by frol** [comment]
@srikanthccv I use this sample repo!

https://github.com/SigNoz/sample-rust-app/blob/c9b0c2a24579915b3ad6ff87fa64c0ff03cc92fe/Cargo.toml#L14

**Comment by srikanthccv** [comment]
@forl there is very limited number of instrumentations supported by OpenTelemetry for rust crates. I don't think there are any instrumentation crates for popular db packages so there is no way to automatically trace them as of now. Here you can find the full list https://github.com/open-telemetry/opentelemetry-rust#related-crates.

**Comment by barakb** [comment]
Are you going to update this example to latest rust otel creates ?
